### ISSUE TITLE
Add alt text for figures

### ISSUE THREAD

**Issue by domoritz** [typst/typst#3701]
### Description

There is a discussion about accessibility in #133 but even if the alt text is not used yet, it would be nice to have alt text support in the syntax so that once the feature lands in the compiler, newly compiled files pick up the alt texts. 

### Use Case

Alt texts are essential for screen reader users and in general are a best practice.

**Comment by laurmaedje** [comment]
Images already have an alt text attribute. Do you want alt text also for other kinds of figures?

**Comment by domoritz** [comment]
Nah, silly me looked at figures not images for alt text.

I'm looking forward to when they will be used to generate accessible document like html and pdf.
